server.js

```diff
@@ -241,16 +252,16 @@ async function addParticipantToSession(account_id, participant_id, session_id) {
  */
 async function initiateCallToPSTN(account_id, from_number, to_number) {
   // call body, see here for more details: https://dev.bandwidth.com/voice/methods/calls/postCalls.html
-  var body = new BandwidthVoice.ApiCreateCallRequest({
+  var body = {
     from: from_number,
     to: to_number,
     applicationId: process.env.BW_VOICE_APPLICATION_ID,
     answerUrl: process.env.BASE_CALLBACK_URL + "callAnswered",
     answerMethod: "POST",
     callTimeout: "30",
-  });
+  };
 
-  return await voiceController.createCall(accountId, body);
+  return await voiceController.createCall(account_id, body);
```

Comment:
I have nothing against having the underscore, but any reason why we are going from a camelCase to this? This is pretty python-ey to me

Reply:
I did this to distinguish between this function arg and the member/instance variable. We really don't need both the function arg and the instance variable, so I'm happy to remove one of them. I'd lean toward removing the function arg. Thoughts?

Reply:
I removed passage of the account ID as a function arg, and made all variables camel case

Reply:
sounds good!